http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -901,7 +909,7 @@ class GKEInterpreter[F[_]](
               config.monitorConfig.startApp.interval
             ).interruptAfter(config.monitorConfig.startApp.interruptAfter).compile.lastOrError
           } yield last.isDone
-        case _ =>
+        case AppType.Wds | AppType.HailBatch =>
```

Comment:
I wanna call this out. We should avoid using `_` ideally since it can easily hide bugs. Making this explicit forces developer to think about addressing each case specifically and be intentional about it